packages/eslint-config-algolia/vue.js

```diff
@@ -19,15 +19,8 @@ module.exports = {
       jsx: true,
     },
   },
-  extends: [
-    'plugin:vue/strongly-recommended',
-    'eslint:recommended',
-    'plugin:import/errors',
-    './rules/base.js',
-    'prettier',
```

Comment:
prettier seems to be removed now

Reply:
one of the goal of this PR is to be able to combine extends and avoir redeclaration, as the doc changes suggest prettier is now in base `extends: ['algolia', 'algolia/vue']`